test/e2e/service_bootstrap.py

```diff
@@ -16,12 +16,65 @@
 
 from acktest.bootstrapping import Resources, BootstrapFailureException
 from e2e import bootstrap_directory
+from acktest.bootstrapping.s3 import Bucket
 from e2e.bootstrap_resources import BootstrapResources
+from acktest.resources import random_suffix_name
+from e2e.common import TRAIL_NAME
+from acktest.aws.identity import get_region, get_account_id
+
+BUCKET_NAME = random_suffix_name("trail-bucket-", 24)
+
+BUCKET_POLICY_FOR_CLOUDTRAIL = """{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "AWSCloudTrailAclCheck20150319",
+            "Effect": "Allow",
+            "Principal": {"Service": "cloudtrail.amazonaws.com"},
+            "Action": "s3:GetBucketAcl",
+            "Resource": "arn:aws:s3:::$NAME",
+            "Condition": {
+                "StringEquals": {
+                    "aws:SourceArn": "arn:aws:cloudtrail:$REGION:$ACCOUNT_ID:trail/$TRAIL_NAME"
+                }
+            }
+        },
+        {
+            "Sid": "AWSCloudTrailWrite20150319",
+            "Effect": "Allow",
+            "Principal": {"Service": "cloudtrail.amazonaws.com"},
+            "Action": "s3:PutObject",
+            "Resource": "arn:aws:s3:::$NAME/AWSLogs/$ACCOUNT_ID/*",
+            "Condition": {
+                "StringEquals": {
+                    "s3:x-amz-acl": "bucket-owner-full-control",
+                    "aws:SourceArn": "arn:aws:cloudtrail:$REGION:$ACCOUNT_ID:trail/$TRAIL_NAME"
+                }
+            }
+        }
+    ]
+}""".replace(
+        "$TRAIL_NAME", TRAIL_NAME,
+    ).replace(
+        "$REGION", get_region(),
+    ).replace(
+        "$ACCOUNT_ID", get_account_id(),
+    ).replace(
+        "$NAME", BUCKET_NAME,
+    )
```

Comment:
This could be made cleaner with a [multi-line f-string](https://stackoverflow.com/a/58090379/1278784)

Reply:
I was looking for this! thanks :)